tools/docker/22.04/Dockerfile

```diff
@@ -199,7 +199,7 @@ ENV HDF5_PLUGIN_PATH /usr/local/lib/python3.10/dist-packages/hdf5plugin/plugins
 ENV KOTEKAN_BUILD_DIRNAME build-2204
 
 # Update Julia...
-RUN juliaup update
+RUN juliaup update && echo 2
```

Comment:
Why `echo 2`?   Was this just to force a change?

Reply:
Yep!  The CI pipelines need a bit of work to handle the 'chord' branch analogously to the way they handle the 'develop' branch.